torchao/prototype/custom_fp_utils.py

```diff
@@ -24,14 +26,24 @@ def _n_ones(n: int) -> int:
 F32_EXP_BIAS = _n_ones(EBITS_F32 - 1)
 
 
-def _f32_to_fpx_unpacked(x: Tensor, ebits: int, mbits: int) -> Tensor:
+class RoundingMode(Enum):
```

Comment:
optional nit: a comment for more info on stochastic rounding would be great for future code readers